startups/lib/main_0.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:english_words/english_words.dart';
```

Comment:
In the end, I think that `pkg:english_words` provides the best combination of simplicity and usability (and fun).